stdlib/char.gr

```diff
@@ -196,8 +208,8 @@ export let rec toString = (c: Char) => {
   }
   let str = allocateString(n)
   Memory.copy(str + 8n, c + 4n, n)
-  let ret = WasmI32.toGrain(str): String
+  let ret = WasmI32.toGrain(str): (String)
   Memory.decRef(WasmI32.fromGrain(c))
   Memory.decRef(WasmI32.fromGrain(toString))
   ret
-}
+}
```

Comment:
Yeah, looks like it's that on all the files.

Reply:
Yup, and fixed.